CMakeLists.txt

```diff
@@ -303,8 +302,15 @@ set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} ${LINKER_FLAGS}")
 add_library(stdlib STATIC static_libs/chaiscript_stdlib.cpp)
 add_library(parser STATIC static_libs/chaiscript_parser.cpp)
 
-add_library(chaiscript_stdlib-${CHAI_VERSION} MODULE src/chaiscript_stdlib_module.cpp)
-target_link_libraries(chaiscript_stdlib-${CHAI_VERSION} ${LIBS} ${CMAKE_THREAD_LIBS_INIT})
+add_library(chaiscript MODULE src/chaiscript_stdlib_module.cpp)
+set_target_properties(chaiscript PROPERTIES 
+                                 OUTPUT_NAME chaiscript_stdlib-${CHAI_VERSION}
+                     )
+target_link_libraries(chaiscript ${LIBS})
+if ( !CMAKE_HOST_WIN32 )
```

Comment:
There is no `!` operator in CMake. Also `Threads::Threads` should just work on Windows.

Reply:
I'll correct the not (!) or eliminate if needed;  I seem to remember trying Threads::Threads on Windows and having issues.  I will revisit.